### ISSUE TITLE
Could pitch bend range expand beyond 2 semitones?

### ISSUE THREAD

**Issue by brickblock369** [Birch-san/juicysfplugin#59]
I'm using FL Studio to run JuicySF, and no matter how much I change the regular pitch bend range value, or the "send pitch bend range" in FL's plugin wrapper settings, the only change I notice is the amount that changes when shifting the knob, but the pitch bend range is still locked to 2 semitones.

I can't tell if this is the FL Studio's issue with VSTs, or the plugin's issue. <details><summary>I've attempted to try it on other hosts but they failed to run the plugin:</summary>The hosts are LMMS and OpenMPT, but LMMS does not support VST3 for the moment and OpenMPT just has an error when loading the plugin file.</details>

**Comment by DaforLynx** [comment]
Yes, it uses Fluidsynth so there is full compatibility with that. What you need to do is set CCs 100 and 101 to 0, and then set CC 6 to the range in semitones you want.

**Comment by brickblock369** [comment]
Thank you, that solved it.
